go/vt/schemamanager/schemaswap/schema_swap.go

```diff
@@ -106,26 +106,24 @@ type shardSwapMetadata struct {
 // input argument is the SQL statements that comprise the schema change that needs to be
 // pushed using the schema swap process.
 func (schemaSwap *Swap) Run(sql string) error {
-	err := schemaSwap.createShardObjects()
-	if err != nil {
+	if err := schemaSwap.createShardObjects(); err != nil {
 		return err
 	}
-	err = schemaSwap.initializeSwap(sql)
-	if err != nil {
+	if err := schemaSwap.initializeSwap(sql); err != nil {
 		return err
 	}
-	err = schemaSwap.runOnAllShards(
+	errHealthWatchers := schemaSwap.runOnAllShards(
 		func(shard *shardSchemaSwap) error {
 			return shard.startHealthWatchers()
 		})
 	// Note: this defer statement is before the error is checked because some shards may
 	// succeed while others fail. We should try to stop health watching on all shards no
 	// matter if there was some failure or not.
 	defer schemaSwap.stopAllHealthWatchers()
-	if err != nil {
-		return err
+	if errHealthWatchers != nil {
+		return errHealthWatchers
 	}
-	err = schemaSwap.runOnAllShards(
+	err := schemaSwap.runOnAllShards(
```

Comment:
That will make "if" condition spread over 4 lines and blend with the code inside "if" due to the same indentation. Are you sure that needs to be done?